presentations/proposed/scipy2022/general/proposal.md

```diff
@@ -5,30 +5,34 @@ Jul 11-17, 2022
 
 
-## Title
-
+## Scientific Python - From GitHub to TikTok
 
 
 ## Short Summary
 
-<!--
-The brief description which will appear in the online program and give
-attendees a basic sense of your talk. This should be around 100 words or less.
--->
+The Scientific Python project aims to better coordinate the ecosystem and prepare Scientific Python for the next decade. We are a group of volunteers behind several projects from the ecosystem committed to expanding our community by generating a welcoming and friendly environment where projects collaborate and build together.
+
 
 
 ## Abstract
 
-<!--
-Your placement in the program will be based on reviews of your abstract.
-This should be a roughly 500 word outline of your presentation. This outline
-should concisely describe software of interest to the SciPy community, tools or
-techniques for more effective computing, or how scientific Python was applied
-to solve a research problem. A traditional background/motivation, methods,
-results, and conclusion structure is encouraged but not required. Links to
-project websites, source code repositories, figures, full papers, and evidence
-of public speaking ability are encouraged.
--->
+As developers, sometimes we focus too much on writing code and too little in community. However, open source is about more than that. It is about communicating, teaching and collaborating. The Scientific Python project’s goal is to unify the community by promoting integration, diversity and wide participation, while generating resources that help bring contributors, developers and users together. 
+
+### Documentation
+
+Documentation is a key aspect of every Python software project. Thus, we want to have a central landing place for information that will help users and contributors better understand the ecosystem. The scientific-python.org website contains resources that we hope are adopted by most of the projects. They include videos that orient new people to the ecosystem, interviews that introduce some of the members of our community and how they contribute to it, and demos, which give an overview of the problem solving workflow in the Python ecosystem. 
+
+### Blog 
+
+The [Scientific Python blog](http://blog.scientific-python.org/) is a place where contributors can share their thoughts and ideas in order to achieve collaborative learning. We strongly believe that healthy communities are built when everyone’s voice is heard and different perspectives are valued.
+
+### Social Media
+
+Community outreach and engagement are also part of growing our community. Nowadays, Facebook, Twitter, Instagram and TikTok are a big part of people's lives and as time passes by, communications change and people learn and collaborate differently. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us diversify our community and stay in touch with more people that are interested in contributing.
```

Comment:
```suggestion
Outreach and engagement are also important ways of growing our community. Nowadays, Facebook, Twitter, Instagram, and TikTok are a big part of people's lives and as time passes by, communications change and people learn and collaborate differently. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us diversify our community and stay in touch with more people that are interested in contributing.
```

I'm not quite sure I follow the point you are trying to make with "as time passes by, communications change and people learn and collaborate differently".

Reply:
I would just delete the phrase "as time passes by, communications change and people learn and collaborate differently".